### ISSUE TITLE
use python-pandas for statistics

### ISSUE THREAD

**Issue by jkall** [jkall/qgis-midvatten-plugin#178]
The python module "pandas" include a lot of regression analysis tools (and other statistics) that would be very useful in the Midvatten plugin. It would be nice to have a separate Midvatten menu entrance for "analyst", "regressions and statistics" or similar from which a number of analyst features could be activated.

The python-pandas module is easily installed on nix-platforms.  And it also seems that pandas can be installed quite easily on win platform if using pip install (an osgeo4w installation of qgis is assumed):
http://lists.osgeo.org/pipermail/qgis-developer/2016-November/045882.html

related issues: 
https://github.com/jkall/qgis-midvatten-plugin/issues/100
https://github.com/jkall/qgis-midvatten-plugin/issues/101

**Comment by jkall** [comment]
It seems python-pandas may be installed through osgeo4w:
https://trac.osgeo.org/osgeo4w/ticket/472
Does it make pyton-pandas available to qgis?

**Comment by HenrikSpa** [comment]
I've not yet tried the above comment, but this method worked in my windows:

http://gis.stackexchange.com/questions/98214/how-to-get-qgis-to-play-with-pandas-windows
"I'm not using the OSGeo4W package but rather the standalone QGIS latest release. I was able to import Pandas into the QGIS python instance by taking the following simple steps. Perhaps the same will work for you or with some minor tweaks.

Install Anaconda: this has the packaged Pandas library readily available in the site-packages folder.
Copy paste the Pandas folder from the Anaconda site-packages and paste into your QGIS site-packages folder. I also included the pandas egg-info file.
At this point I thought it would work but the import was throwing an error about not finding pkg_resources.py. I found that and the .pyc in the Anaconda pip folder and pasted both in the QGIS site-packages folder.
After taking the above steps, I was able to cleanly import Pandas into my PyQGIS environment without any problems."

So in other words:
Install anaconda version Anaconda-2.0.0-Windows-x86_64.exe 334.4M 2014-05-28 16:50:53 9ec65c4cc0d640ff36f89193cb9e7b7d
Found at https://repo.continuum.io/archive/index.html

Use the quoted instructions above and then copy the folders statsmodels and patsy in the same way.
